index.js

```diff
@@ -36,17 +36,13 @@ const { spawn } = require('child_process');
     '-f', 'image2pipe',
     '-r', '30',
     '-i', '-',
-    //'-f', 'alsa', '-i', 'default', // Adjust audio input as needed
     '-c:v', 'libx264',
     '-preset', 'veryfast',
     '-b:v', '3000k',
     '-maxrate', '3000k',
     '-bufsize', '6000k',
-    //'-c:a', 'aac',
-    //'-b:a', '128k',
     '-f', 'mpegts',
-    'udp://127.0.0.1:1234?pkt_size=1316'
-    //'srt://listener_ip:port?pkt_size=1316' // Replace with your SRT listener details
+    process.env.STREAM_DEST
   ]);
 
```

Comment:
(This regards line 73, which GitHub won't let me select :()

1. Have you considered using `setInterval`?
2. Either way, it would be safest/cleanest to also save the result of `setTimeout` or `setInterval` in a variable which is then cleared when the function exits - otherwise I am not sure if this doesn't cause `captureFrame` to continuously execute even after the `ffmpeg` process has exited.
3. Also - I am not really that familiar with the logic of what happens if frame delays start piling up.
If this becomes an issue at any point, it might make sense to use a more specific method like [`lodash`'s debounce/throttle](https://css-tricks.com/debouncing-throttling-explained-examples/) or [`requestAnimationFrame` ](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestAnimationFrame), but I leave this to your judgement.